### ISSUE TITLE
VEDAI dataset

### ISSUE THREAD

**Issue by passingdragon** [yemu1138178251/FFCA-YOLO#13]
May I ask how the training set and the test set of VEDAI data set are divided? Can you provide the processed VEDAI data set? Thank you!

**Comment by Godk02** [comment]
不是已经划分好了吗

**Comment by Godk02** [comment]
问了这一篇的作者
SuperYOLO: Super Resolution Assisted Object
Detection in Multimodal Remote Sensing Imagery
作者提供了下载并且划分好了，问了作者说按照划分好的10份，跑10次取平均
